src/transformers/onnx/convert.py

```diff
@@ -25,6 +26,8 @@
 from transformers.onnx.config import OnnxConfig
 from transformers.utils import logging
 
+from ..feature_extraction_utils import FeatureExtractionMixin
```

Comment:
Same here. All the imports are a mess of relative and absolute. This should all be relative.